doc/ElementsApplicationPattern/Principles/ConfigFile/ConfigFile.md

```diff
@@ -1,11 +1,20 @@
-## CONFIGfile 
-The values defined in a CONFIGfile indicate the initial state of the application as soon as it is installed. This can be customized if require.
-The CONFIGfile provides information about , 
+# CONFIGfile  
 
+The values defined in a CONFIGfile indicate the initial state of the application as soon as it is installed.  
+It can be customized if require.  
+Changes to the configuration of the application, which are happening during operation of the application, are also written into the CONFIGfile.  
+So, in case of failure, a new instance of the application could be started with the exact latest configuration of the broker instance.  
```

Comment:
can we change the following ?
from 
broker instance
to 
broken instance

Reply:
Yes, we can.